gaudi/Gaudi-SPI-Firmware-Upgrade.md

```diff
@@ -37,7 +38,7 @@ $ oc logs gaudi-spi-firmware-upgrade-ndmjp
 ####
 #### Finished sending firmware: OK
 ```
-Verify using `hl-smi` tool: 
+Verify by following the [firmware version check](https://github.com/intel/intel-technology-enabling-for-openshift/tree/main/tests/gaudi/l2#check-firmware-version-with-hl-smi). 
 ```

```

Comment:
I think we can remove line 42 to line 52. :-)

Reply:
updated